src/cv/settings.py

```diff
@@ -21,6 +21,12 @@
 RECOGNIZER = WORKDIR + os.environ.get('RECOGNIZER_PATH')
 LABEL_ENCODER = WORKDIR + os.environ.get('LABEL_ENCODER_PATH')
 TRACK_TIME = 600
+"""
+  Directories
+"""
+UPLOAD_FOLDER_PATH = '/tmp'
+ALLOWED_EXTENSIONS = {'txt', 'pdf', 'png', 'jpg', 'jpeg', 'gif'}
```

Comment:
I don't see this used

Reply:
It'll be an upcoming pr to restrict the file types on the server

Reply:
I removed it for now